reports/issue-count-cycle-time.ts

```diff
@@ -0,0 +1,124 @@
+import {ProjectData, ProjectIssue, IssueCardEventProject} from '../interfaces';
+import * as rptLib from '../project-reports-lib';
+const tablemark = require('tablemark')
+import * as os from 'os';
+
+const reportType = 'project';
+export {reportType};
+
+export type CycleTimeData = { [key: string]: CycleTimeStageData }
+interface CycleTimeStageData {
+  title: string,
+  cycletime: number,
+  limit: number,
+  count: number,
+  flag: boolean
+}
+
+interface CycleTimeRow {
+    worktype: string,
```

Comment:
the worktype and labels seems redundant to me (when I looked at it visually).  I think the labels is enough??